docs/METRICS.md

```diff
@@ -258,10 +229,18 @@ This is stuff we get from including ga.js on Screenshots pages.
 
 ### Database Metrics
 
-We have a pretty rich database in Screenshots, and we can do all kinds of queries on the database.  These might include:
+We can do queries directly on the database of Firefox Screenshots.  These might include:
 
 1. Look at cohorts of individuals, reviewing their shot creation patterns
 2. How many people make a couple shots?  How many make a lot of shots?
 3. See if people are making shots of mostly private or mostly public pages
 4. Identify the dimensions of shots
-5. Compare shot URLs to a domain whitelist, to determine broad categories of sites that are popular to view
+5. Compare shot URLs to a domain allowlist, to determine broad categories of sites that are popular to view
+
+### Opt-out
+
+The add-on reads the Telemetry opt-out preference (`toolkit.telemetry.enabled`) before sending any metrics data to the servers.  If this preference is false, or if there is any issue trying to fetch the preference, then no data is sent.
```

Comment:
I thought their might be other entry points. Like, if a person hits the keyboard combo or if we eventually provide some other link (a my shots button?)